src/Headers/ContentSecurityPolicy/Header.php

```diff
@@ -0,0 +1,83 @@
+<?php declare(strict_types = 1);
+
+namespace Pd\SecurityHeaders\Headers\ContentSecurityPolicy;
+
+final class Header implements \Pd\SecurityHeaders\Headers\IHeader
+{
+
+	/**
+	 * @var Directive|null
+	 */
+	private $defaultSrc;
+
+	/**
+	 * @var array|Directive[]
+	 */
+	private $directives = [];
+
+	/**
+	 * @var bool
+	 */
+	private $blockAllMixedContent;
+
+
+	public function getName(): string
+	{
+		return 'Content-Security-Policy';
+	}
+
+
+	public function getValue(): string
+	{
+		$directives[] = $this->defaultSrc ? ('default-src ' . $this->defaultSrc) : '';
```

Comment:
Proč máš `default-src` zvlášť v `$this->$defaultSrc` a neukládáš to rovnou do pole `$this->directives` jako ostatní direktivy?

Reply:
Abych zaručil, že bude vypsaná na prvním místě.

Reply:
A to tak musí být nebo jsi to tak udělal jenom aby bylo hned vidět, jaké jsou výchozí zdroje?

Reply:
Abych to viděl já v té hlavičce, podle mě to nemusí být na prvním místě.

Reply:
Ok, proč ne.